apps/src/code-studio/initApp/loadApp.js

```diff
@@ -153,4 +337,84 @@ module.exports = function (callback) {
   } else {
     loadLastAttemptFromSessionStorage();
   }
+}
+
+window.dashboard = window.dashboard || {};
+window.dashboard.project = project;
+
+window.apps = {
```

Comment:
If we're introducing a new global, can we have  a clear story about what does/does not belong on here? (With that story either told here, or in a doc linked to in a comment here)